### ISSUE TITLE
use of undeclared crate or module `tonic`

### ISSUE THREAD

**Issue by apianmuse** [Fluidex/plonkit#56]
I get an error When I install plonkit with the following command
```
$ cargo install --git https://github.com/Fluidex/plonkit

error[E0433]: failed to resolve: use of undeclared crate or module `tonic`
  --> src/lib.rs:20:5
   |
20 |     tonic::include_proto!("plonkitserver");
   |     ^^^^^ use of undeclared crate or module `tonic`

```

How can i solve it?

**Comment by HAOYUatHZ** [comment]
Thanks for your feedback! Our CI compile plonkit with `--all-features` enabled so it didn't detect this issue.

I have fix this problem in https://github.com/Fluidex/plonkit/pull/57. Hope it works now!

Let me know if you have further problems.

**Comment by apianmuse** [comment]
> Thanks for your feedback! Our CI compiles plonkit with `--all-features` enabled so it didn't manage to detect this issue.
> 
> I have fix this problem in #57. Hope it works now!
> 
> Let me know if you have further problems.

Thank you!!
